### DISCUSSION TITLE
Libpostal and Spring Boot - need help to run

### DISCUSSION THREAD

**Discussion by PiotrKotynia** [bytedeco/javacpp-presets#1285] [Q&A] [answered]
Dears,

Hello! I'm glad that I can ask a question to this prominent community!

I'm quite fresh to those topics and I would need some guidance how to run the library of Libpostal within Java Spring Boot application.
I've added the dependencies according to the read.me file and also downloaded the dataset via MSYS2 (Win11) but the code below does not work. Basically because it does not find the dataset but I have no idea how to save the dataset on WIn11 to "normal" path like "D:/Temp" on my disc as MSYS2 can only download that to some virtual Linux location like "/home".
Can you help me? How can I save the dataset localy to "D:/Temp" on Windows machine instead of some virtual location from MSYS2 ("/home")?
The code below:
String dataDir = "D:\\Temp";
String libpostal_data = Loader.load(org.bytedeco.libpostal.libpostal_data.class);
ProcessBuilder pb = new ProcessBuilder("bash", libpostal_data, "download", "all", dataDir);

Thanks in advance for your help and I would be glad to provide you more information if necessary (I'm just not fully aware what info would be useful for you so that this question might seem to be inaccurate).

Kind regards,
Piotr

**Comment by saudet** [comment, marked as answer]
You can run that bash command outside of Java if it's easier for your system, that's fine too.

**Reply by saudet** [reply]
And I'm pretty sure the script accepts paths like "D:/Temp" just fine. Just try to give it that, and it should work.

**Reply by saudet** [reply]
If not, MSYS2 should provide access to the C: and D: drives via /c/ and /d/, see https://www.msys2.org/docs/filesystem-paths/

**Reply by PiotrKotynia** [reply]
@saudet thanks a lot for a hint! We have a small pause on this project right now so that I will have to stop trials temporarily. You helped a lot!
